pkgs/test_core/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   source_span: '>=1.8.0-nullsafety <1.8.0'
   stack_trace: '>=1.10.0-nullsafety <1.10.0'
   stream_channel: ">=2.1.0-nullsafety <2.1.0"
-  vm_service: '>=1.0.0 <6.0.0'
+  vm_service: '>=6.0.0-nullsafety-dev <6.0.0'
```

Comment:
What is up with the `-dev` here? Are we expecting breaking changes coming to this release, because we can't really support it in this package then if we could be broken at any time...

Reply:
@bkonyi

Reply:
This isn't quite ready to be rolled. Waiting on some initial testing from @grouma and @kenzieschmoll to confirm we don't break anything in DWDS and DevTools, at which point I'll publish a non-dev version.

Reply:
Ok. I'll close this for now then to get it off review queues.

In that case, flutter is still blocked on migrating driver/integration test (/cc @goderbauer @Hixie FYI)